source/puppet/5.5/modules_documentation.markdown

```diff
@@ -194,116 +194,103 @@ To transition your module to the `REFERENCE.md` model, add Strings comments to y
+If you're not yet ready to use Puppet Strings to generate your reference docs, you can manually create a `REFERENCE.md` file listing all of your module's classes, defined types, functions, tasks, task plans, and resource types and providers. This is useful if you are in the process of moving your reference documentation into code comments, or if your module contains resources that aren't yet supported in Strings.
 
-To manually document reference information, start your reference information with a small table of contents that first lists the classes, defined types, and resource types of your module. If your module contains both public and private classes or defined types, list the public and the private separately. Include a brief description of what these items do in your module.
+If your module contains a `REFERENCE.md` file, the Forge displays the contents in a Reference tab on the module's detail page, whether you created the file manually or with Strings. Write the `REFERENCE.md` file in Markdown, and keep it in the root directory of your module.
 
-```
-## Reference
-
-### Classes
-
-#### Public classes
-
-*[`pet::cat`](#petcat): Installs and configures a cat in your environment.
+If you already have a README with an accurate reference section, you can remove that section from your README and paste it into a `REFERENCE.md` file. The formatting isn't vastly different.
 
-#### Private classes
+> **Important**: If you create a `REFERENCE.md` manually, and then generate Markdown with Puppet Strings, it will overwrite your `REFERENCE.md` unless you give the files different name. The module detail page on the Forge displays only the `README.md` and `REFERENCE.md` files.
 
-*[`pet::cat::install`]: Handles the cat packages.
-*[`pet::cat::configure`]: Handles the configuration file.
-```
+At the top of your `REFERENCE.md`, include a ` # Reference` header and a table of contents.
 
-After this table of contents, list the parameters, providers, or features for each element (class, defined type, function, and so on) of your module. Be sure to include valid or acceptable values and any defaults that apply.
+In the table of contents, list each element of your module: classes, defined types, functions, tasks, task plans, and resource types and providers. Provide brief descriptions, and separate public and private classes.
 
-Each element in this list should include:
+Include the following information:
 
   * The data type, if applicable.
   * A description of what the element does.
   * Valid values, if the data type doesn't make it obvious.
   * Default value, if any.
 
+```markdown
+# Reference
+
+## Classes
+
+### Public Classes
+*[`cat`](#::cat)
```

Comment:
unless :: is in the heading, it doesn't need to be included in the link, this can just be (#cat). i'm just afraid this could be misinterpreted as a need to incorporate namespacing into your documentation :)

Reply:
Thanks! It's old and leftover; I'll fix it.